include/sys/util.h

```diff
@@ -382,6 +450,25 @@ u8_t u8_to_dec(char *buf, u8_t buflen, u8_t value);
  */
 #define __GET_ARG2_DEBRACKET(ignore_this, val, ...) __DEBRACKET val
 
+/**
+ * @breif Empty macro
+ *
+ * This trivial definition is provided to use where macro is expected
+ * to evaluate to empty replacement string aor when it is needed to
+ * cheat checkpatch.
+ *
+ * Examples
+ *
+ *	#define LIST_ITEM(n) , item##n
+ *
+ * would cause error with checkpatch, but:
+ *
+ *	#define LIST_TIEM(n) EMPTY, item##m
+ *
+ * would not.
+ */
+#define EMPTY
```

Comment:
there is `UTIL_EMPTY(...)` already defined. Could that be reused?

Reply:
Yes! Missed it due to ...

Reply:
Wiat, no. Because to be C99 compliant ... macros should be given at least one param. , I do not think that we are compliant with that in many macros. Lets wait for others to have their insights on that macro.

Reply:
ok, then keep it like it is. Would be good to get that in.